packages/backend/README.md

```diff
@@ -19,12 +22,35 @@ First, install the AWS CLI ([instructions](https://docs.aws.amazon.com/cli/lates
 
 ### 2. Store private keys in AWS Secrets Manager
 
-Run `key-management.sh` script to store your Alchemy API key and Ethereum Executor Account private key in AWS Secrets Manager:
+Run `key-management.sh` script to store:
+
+1. Alchemy API key
+2. Ethereum Executor Account private key
+3. Google Cloud Project API key
+4. Google Cloud Project Id
+5. ReCaptcha key (Score based)
+6. ReCaptcha Action specified in ReCaptcha call
+7. ReCaptcha threshold (0.0-1.0)*
+
+in AWS Secrets Manager:
 
 ```
 ./key-management.sh --set --alchemy "<YOUR_PRIVATE_KEY>"
 ./key-management.sh --set --executor "0x<YOUR_PRIVATE_KEY>"
+./key-management.sh --set --google-api-key "<YOUR_API_KEY>"
+./key-management.sh --set --google-project-id "<YOUR_PROJECT_ID>"
+./key-management.sh --set --recaptcha-key "<YOUR_RECAPTCHA_KEY>"
+./key-management.sh --set --recaptcha-action "<YOUR_RECAPTCHA_ACTION>"
+./key-management.sh --set --recaptcha-threshold "<YOUR_RECAPTCHA_THRESHOLD>"
```

Comment:
See above

Reply:
It's now provided as an ENV variable to the lambda function and specified as SAM Parameters.